pkg/to/value.go

```diff
@@ -28,6 +28,11 @@ func String(s *string) string {
 	return ""
 }
 
+// StringPtr returns a pointer to the string passed in.
+func StringPtr(s string) *string {
```

Comment:
Thanks for this addition.

Reply:
Added a test for this function too

Reply:
@ytimocin, @rynowak perhaps you can confirm but looking at functions in this package, seems like it is intended to work with input pointers and convert to their respective values and not vice versa. 
If that's the case, I would suggest using existing function to.Ptr() instead defined [here](https://github.com/radius-project/radius/blob/53e19b23e0736161da7804de0a297cefe13f6a71/pkg/to/ptr.go#L22)

Reply:
I agree with @lakshmimsft here. I don't think we need this function. You can see the usages of **to.Ptr** here: https://github.com/search?q=repo%3Aradius-project%2Fradius%20%22to.Ptr%22&type=code.